_data/sites.json

```
@@ -10489,6 +10499,17 @@
         ]
     },
 
+    {
+        "name": "Packt",
+        "url": "https://www.packtpub.com/",
+        "difficulty": "hard",
+        "email": "subscription.support@packtpub.com",
+        "notes": "Send an email to subscription.support@packtpub.com requesting deletion. If you have a subscription plan or free trial be sure to cancel it.",
```

```suggestion
        "notes": "Send an email to them requesting deletion. If you have a subscription plan or free trial be sure to cancel it.",
```
Let's leave the email to its field and simplify the notes